# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/ksendfax/includefix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN ksendfax-3.0.2/ksendfax/myscandialog.cpp ksendfax-3.0.2-new/ksendfax/myscandialog.cpp
--- ksendfax-3.0.2/ksendfax/myscandialog.cpp	2001-11-08 17:00:26.000000000 +0100
+++ ksendfax-3.0.2-new/ksendfax/myscandialog.cpp	2005-05-22 21:18:57.000000000 +0200
@@ -11,6 +11,7 @@
 #include <qstrlist.h>
 #include <qtooltip.h>
 #include <qsizepolicy.h> 
+#include <qcheckbox.h>
 
 #include <kglobal.h>
 #include <klocale.h>
